src/types/globals.ts

```diff
@@ -0,0 +1,13 @@
+//CloudFlare environment variables
+declare global {
+  const WEBHOOK: string;
+  const SECRET: string;
+  const OPENAI_API_KEY: string;
+  const SUPABASE_URL: string;
+  const SUPABASE_KEY: string;
+  const GITHUB_PAT: string;
+  const INSTALLATION_TOKEN: string;
+  const TOKEN: string;
+}
+
+export {};
```

Comment:
Why do we need these globals? can't we read directly from environment.json?

Reply:
> Why do we need these globals? can't we read directly from environment.json?

We read directly, these are global typings to make calling env the cloudflare worker way valid on TS